src/Series.cpp

```diff
@@ -502,15 +502,18 @@ namespace
         int padding;
         uint64_t iterationIndex;
         std::set<int> paddings;
-        for (auto const &entry : auxiliary::list_directory(directory))
```

Comment:
This fixes another subtle bug: Creating a Series via Append mode should be possible. But so far, this routine failed if the directory was not present.